build-livecd-root

```diff
@@ -1,95 +1,10 @@
 #!/bin/bash -x
 
-# the portion of building a livecd that needs to be run as root
-# run this after running build-livecd
-#
-# The temp dir in which things happen is by default deleted when this
-# script ends, unless an error happens or the KEEP_TMPDIR variable is set
-
-function cleanup() {
-    if [ -n "$KEEP_TMPDIR" ]; then
-        echo "Not removing tmpdir $tmpdir"
-    else
-        [ -d $tmpdir ] && rm -rf $tmpdir
-    fi
-}
-
-if [ $(id -u) != 0 ]; then
-  echo "Please run this script as root"
-  exit 1
+rm /var/tmp/lmc-work-* -rf
+if [ ! -e CentOS-8.4.2105-x86_64-dvd1.iso ]; then
+    wget http://ftp.osuosl.org/pub/centos/8.4.2105/isos/x86_64/CentOS-8.4.2105-x86_64-dvd1.iso
 fi
```

Comment:
Allright, this is a big change. Previously the script was building from live repos, this change requires the full DVD and then only installs minimal system from it. On our CI this will require more space.

Not that I dislike it, I am just trying to summarize what this can possibly affect. Another thing to consider is 8.4 (or 8.5 not sure) is the last CentOS 8 release, so do we want to use CentOS Stream? It appears to have a isodirect redirector to the "latest" DVD:

wget http://isoredirect.centos.org/centos/8-stream/isos/x86_64/CentOS-Stream-8-x86_64-latest-dvd1.iso

The problem with that is that it changes quite often, I currently see a build that is one day old.

My suggestion:

* name the ISO as `install.iso` or something (can be a symlink too)
* let's parametrize the ISO URL
* let's use http://isoredirect.centos.org/centos/8-stream/isos/x86_64/CentOS-Stream-8-x86_64-latest-dvd1.iso as the default value
* for development builds this is fine
* for production builds we will probably keep a copy of DVD and build from that so we can repeat builds (unfortunately CentOS Stream project deletes all old ISOs)
* users can opt-in to building from RHEL, AlmaLinux or other alternatives

What you think?